src/main/java/com/team4/testingsystem/entities/Level.java

```diff
@@ -0,0 +1,28 @@
+package com.team4.testingsystem.entities;
+
+import javax.persistence.*;
+
+@Entity
+public class Level {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(name = "level_name")
+    private String levelName;
```

Comment:
We are in a level class. It already has level context. You don't need to repeat it with a level prefix